docs/book/v1/emitting-responses.md

```diff
@@ -1,5 +1,6 @@
 # Emitting Responses
 
+<!-- markdownlint-disable-next-line MD001 -->
```

Comment:
I was getting a linting error due to this file, so fixed it here, as it was trivial, and easier than doing a separate PR.

Reply:
afaik that ja due to the fact that there is no secondary header. maybe we should add one instead of ignoring the error?

Reply:
No - this is a case where we have a "note", and those are indented at level 3.

Reply:
> …and those are indented at level 3.

…and adding another header with level 2 before will solve the problem. 😉

Reply:
We can do that in a different PR. :)